### ISSUE TITLE
[docs] Add clarifying example of `fn` param to option()

### ISSUE THREAD

**Issue by gregfenton** [tj/commander.js#1909]
The description for what the `fn` parameter of `option()` is sparse and there are no examples of its use.  Looking at the code for it, the use is non-obvious.

**Comment by gregfenton** [comment]
Might also consider adding an example of a parameter being passed that is quoted:
```
$ node options-common.js --pizza-type='veggie lovers' -s

pizza details:
- small pizza size
- veggie lovers
```

See the [Options docs ](https://github.com/tj/commander.js/tree/master#options) and the [options-common.js](https://github.com/tj/commander.js/blob/master/examples/options-common.js) examples.

**Comment by shadowspawn** [comment]
See the README section on custom option processing:

https://github.com/tj/commander.js#custom-option-processing

**Comment by gregfenton** [comment]
Okay, cool that it is in there.  But it was not obvious to see/find.  It comes after a long scroll and after topics labelled "Other" and "Misc"....at which point I assumed I had hit the end of the Options section.

Feel free to close as the docs are there.  But I recommend some sort of tweaking to make this more obvious.  I did not link  the `fn` parameter to be "custom processing".

**Comment by shadowspawn** [comment]
Why were you looking for more about the `fn` parameter? From in-editor assistance while typing, or looking at the Javascript code, or looking at the TypeScript definition?

**Comment by gregfenton** [comment]
In-editor (VSCode).  I was looking to do something new but realized that I didn't have sufficient information to know what `fn` did, or whether it could be a valid approach.  Ultimately chose to do something different, but that was after some frustration trying to understand `fn`.

Again, delighted that the docs are there.  But didn't find those docs on my own.

Thanks!

**Comment by shadowspawn** [comment]
Another person who saw `fn` but not the custom parsing: #1953

**Comment by MaxumCSmith** [comment]
Thank you.

**Comment by shadowspawn** [comment]
Opened #2009 with a rework of the inline documentation. Single most relevant change was renaming `fn` to `parseArg`!

**Comment by shadowspawn** [comment]
Shipped in Commander v11.1.0
